modules/payloads/singles/python/pingback_bind_tcp.rb

```diff
@@ -0,0 +1,61 @@
+require 'msf/core/handler/bind_tcp'
+require 'msf/core/payload/python'
+
+require 'msf/base/sessions/pingback'
+require 'msf/core/payload/pingback'
+
+module MetasploitModule
+
+  CachedSize = 257
+
+  include Msf::Payload::Single
+  include Msf::Payload::Pingback
+  include Msf::Payload::Pingback::Options
+
+  def initialize(info = {})
+    super(merge_info(info,
+      'Name' => 'Python Pingback, Bind TCP (via python)',
+      'Description' => 'Listens for a connection from the attacker, sends a UUID, then terminates',
+      'Author' => 'asoto-r7',
+      'License' => MSF_LICENSE,
+      'Platform' => 'python',
+      'Arch' => ARCH_PYTHON,
+      'Handler' => Msf::Handler::BindTcp,
+      'Session' => Msf::Sessions::Pingback,
+      'PayloadType' => 'python',
+      'Payload' =>
+        {
+          'Offsets' => { },
+          'Payload' => ''
+        }
+      ))
+  end
+
+  def generate
+    super + command_string
+  end
+
+  def command_string
+    self.pingback_uuid ||= self.generate_pingback_uuid
+
+    cmd  = "import socket as s\n"
```

Comment:
This would be better off as a squiggly heredoc: https://github.com/rubocop-hq/ruby-style-guide#squiggly-heredocs